src/renderer/components/swap/Swap.tsx

```diff
@@ -1407,22 +1407,28 @@ export const Swap = ({
 
   // Function to reset the slider to default position
   const resetToDefault = () => {
-    setStreamingInterval(3) // Default position
+    setStreamingInterval(1) // Default position
     setStreamingQuantity(0) // thornode decides the swap quantity
-    setSlider(30)
+    setSlider(26)
     setIsStreaming(true)
   }
 
   // Streaming Interval slider
   const renderStreamerInterval = useMemo(() => {
+    const calculateStreamingInterval = (slider: number) => {
```

Comment:
I preferred the prev version. Forcing the range into position is better than what the pic has. 
![image](https://github.com/asgardex/asgardex-desktop/assets/40199525/8df38e14-42f3-47b1-9abc-b1e46ba537e1)